modules/articles.py

```diff
@@ -0,0 +1,59 @@
+import newspaper
+import re
+
+class article_content():
+
+    def __init__(self, article_object):
+        #Whenever a class object is made, the article is tokenized and the dictionary file is opened as a set.
+        self.tokenized_words = nltk.word_tokenize(article_object.text)
```

Comment:
Have you tested the code? nltk is not imported yet

Reply:
Tested the code.

First line was not copied.